### ISSUE TITLE
Lesion Segmentation

### ISSUE THREAD

**Issue by Eugeniosales** [deeplearningunb/Skincare#8]
The first task refers to the data preprocessing, where the images needs to be splitted in different directories according to its respective disease.

To be accepted, the segmentation must:
- [x] Gather the images in its respective directories
- [x] Split it afterwards into test set and training set
- [x]  Manage and keep the metadata associated in it
